master-qa-35860

kmaria opened the pull request.

robertoDiaz committed:
LPS-81743 Create missing components to migrate attachments from one store to another

robertoDiaz committed:
LPS-81743 Store migration doesn't migrate Adaptive Media generated files

robertoDiaz committed:
LPS-81743 Store migration doesn't migrate previews and thumbnails

robertoDiaz committed:
LPS-81743 SF

adolfopa committed:
LPS-81743 No need to unwrap the dlFileEntry

adolfopa committed:
LPS-81743 Remove unneeded attribute and use field injection everywhere

adolfopa committed:
LPS-81743 Rename class, this is closely related to the stores

adolfopa committed:
LPS-81743 Close input stream

adolfopa committed:
LPS-81743 Remove, not thrown

adolfopa committed:
LPS-81743 This does nothing

adolfopa committed:
LPS-81743 Use lambdas instead of anon inner classes

adolfopa committed:
LPS-81743 Move code into try block

adolfopa committed:
LPS-81743 Auto SF

javiergamarra committed:
APIO-99 custom route classes to specify name/path

javiergamarra committed:
APIO-99 expose new classes

javiergamarra committed:
APIO-99 store form instead of optional

javiergamarra committed:
APIO-99 add path and custom to operations

javiergamarra committed:
APIO-99 add target URL in operations

robertoDiaz committed:
LPS-81743 Create a new object that contains the stores instances during the data migration

javiergamarra committed:
APIO-99 add aliases

javiergamarra committed:
APIO-99 custom endpoint

javiergamarra committed:
APIO-99 collection route representor

javiergamarra committed:
APIO-99 item routes function

javiergamarra committed:
APIO-99 custom endpoint

javiergamarra committed:
APIO-99 adapt tests

javiergamarra committed:
APIO-99 javadocs

javiergamarra committed:
APIO-99 keep existing name behaviour

adolfopa committed:
LPS-81743 All file entries have at least one file version

It is not possible for the list to be empty, so no point on checking
that.

adolfopa committed:
LPS-81743 Query file versions instead of file entries

adolfopa committed:
LPS-81743 This will be done when previews/thumnails are migrated

The `migrateGeneratedFiles` will take care of cleaning up the store.

adolfopa committed:
LPS-81743 Migrate all file versions automatically

javiergamarra committed:
APIO-99 source format and parameter order

javiergamarra committed:
APIO-99 add javadocs and move api methods

javiergamarra committed:
APIO-99 rename

javiergamarra committed:
APIO-99 apply forms

javiergamarra committed:
APIO-99 add page and item tests

adolfopa committed:
LPS-81743 Use the stores in the convert process directly

The file entry migration is already handled by the system, so
implementations should only be necessary for custom uses of the store.

adolfopa committed:
LPS-81743 Make convert processes honor deletion if requested by user

adolfopa committed:
LPS-81743 Implement move operation for adaptive media

adolfopa committed:
LPS-81743 Move migration code into "modular" implementations

adolfopa committed:
LPS-81743 Not used anywhere

adolfopa committed:
LPS-81743 Move logic into store

adolfopa committed:
LPS-81743 Apply changes to other stores

adolfopa committed:
LPS-81743 Don't swallow exceptions

adolfopa committed:
LPS-81743 Use lambdas instead of anon inner classes

adolfopa committed:
LPS-81743 Needs Java 8 features

adolfopa committed:
LPS-81743 Semver

adolfopa committed:
LPS-81743 Update deprecation messages

javiergamarra committed:
APIO-99 adapt tests

javiergamarra committed:
APIO-99 sf

javiergamarra committed:
APIO-99 change types to allow more flexibility implementing forms

javiergamarra committed:
APIO-99 fix missing route exception

javiergamarra committed:
APIO-99 add example in blogs sample app

javiergamarra committed:
APIO-99 adapt tests to new types

javiergamarra committed:
APIO-99 fix target generation and sf

Camork committed:
LPS-84670 Create ext module project template

lanceji92674957 committed:
LPS-84838 Remove confusing logs when upgrade

dantewang committed:
LPS-81670 Portlet 3.0 Async: More accurate exception throwing according to JavaDoc

lanceji92674957 committed:
LPS-84838 Make test with full code coverage

dantewang committed:
LPS-81670 Portlet 3.0 Async: Refine the order of notification and logging for portlet async listeners added after completed

javiergamarra committed:
APIO-232 new API for nested and nested lists in forms

javiergamarra committed:
APIO-232 store new fields

javiergamarra committed:
APIO-232 render new fields in form affordance

javiergamarra committed:
APIO-232 use new API

javiergamarra committed:
APIO-232 calculate values

javiergamarra committed:
APIO-232 new parser to extract nested values

javiergamarra committed:
APIO-232 baseline

tinatian committed:
LPS-81670 The http servlet request we delegate to should be consistent. We should always respect the HttpServlet one.

tinatian committed:
LPS-81670 Should delegate the isAsyncSupported checking to http servlet request

javiergamarra committed:
APIO-232 don't force to pass value

tinatian committed:
LPS-81670 SF, no need to add private method

tinatian committed:
LPS-81670 SF, do checking first

tinatian committed:
LPS-81670 SF, move checking to where it is needed

tinatian committed:
LPS-81670 SF, do simple checking first

javiergamarra committed:
APIO-232 renames

javiergamarra committed:
APIO-232 small tests

javiergamarra committed:
APIO-232 formatJavadoc

dantewang committed:
LPS-81670 Should only throw "if asynchronous processing has not yet been started"; isAsyncStarted returning false can't indicate async has never been started

Commit:
LPS-81743

Commit:
LRQA-35860 Add deletion export import

tinatian committed:
LPS-81670 SF

javiergamarra committed:
APIO-99 baseline

clovisdasilvaneto committed:
LPS-84991 Add autobind-decorator dependency

clovisdasilvaneto committed:
LPS-84991 Adjust modal size

clovisdasilvaneto committed:
LPS-84991 Create listeners in FormRenderer for propagating it to the FormBuilder

clovisdasilvaneto committed:
LPS-84991 Add delete field logic on FormBuilder

LPS-84991 Remove delete field logic from PageRenderer

aline-doleron committed:
LPS-84991 Update snapshot

aline-doleron committed:
LPS-84991 SF

aline-doleron committed:
LPS-84991 Rename event name

javiergamarra committed:
APIO-99 add missing entries to tests bnd

aline-doleron committed:
LPS-84991 Create tests

javiergamarra committed:
APIO-99 add explicit import for sf

javiergamarra committed:
APIO-99 sf

javiergamarra committed:
APIO-99 add exclusions to bnd

Commit:
LPS-84478 Remove unused strings

javiergamarra committed:
APIO-99 remove util project from deployment

javiergamarra committed:
APIO-99 apply new sf

javiergamarra committed:
APIO-99 Revert "remove util project from deployment"

This reverts commit 33b21e1f9bd3fc278b43e518817b98e46eff30b7.

javiergamarra committed:
APIO-99 add missing dependencies in util project

javiergamarra committed:
APIO-232 nested form in forms

Commit:
LPS-84478 Create context in order to make ThemeDisplay available

Commit:
LPS-84478 No longer needed

Commit:
LPS-84478 Use display locale when rendering strings on fields

Commit:
LPS-84478 Fix labels and remove unused method

Commit:
LPS-84478 Fix tests

gamerson committed:
LPS-84670 setup generateSample task

gamerson committed:
LPS-84670 we don't support modules-ext for maven

gamerson committed:
LPS-84670 add tests

Preston-Crary committed:
LPS-85212 Allow javax.sql.DataSource to be published into the service's BundleContext to override the portal's DataSource.

gamerson committed:
LPS-84670 simplify

gamerson committed:
LPS-84670 newline

gamerson committed:
LPS-84670 use published

clovisdasilvaneto committed:
LPS-84295 Adjust pagination style for it appearance be aligned to the old forms

clovisdasilvaneto committed:
LPS-84295 Create pagination feature for allowing user changes the pagination mode

clovisdasilvaneto committed:
LPS-84295 Adjust page container margin and padding

clovisdasilvaneto committed:
LPS-84295 Create test cases to cover the pagination mode feature

clovisdasilvaneto committed:
LPS-84295 Add switch labels on Language.properties

javiergamarra committed:
APIO-99 sf

Commit:
LPS-85261 image-uploader-web - UploadImageMVCActionCommand.java - Display error message for UploadRequestSizeException

mtambara committed:
LPS-85003 Sort

gamerson committed:
LPS-84670 fix newline handling

gamerson committed:
LPS-84670 add compilyOnly configuration files to classpath

gamerson committed:
LPS-84670 switch back to compileOnly

adolfopa committed:
LPS-81513 Update Commons Compress and Tika libs

adolfopa committed:
LPS-81513 PDFBox was already on 2.0.9

adolfopa committed:
LPS-81513 Regen versions-complete.xml and versions.xml

dgarciasarai committed:
LPS-85097 Create user notifications for workflow tasks

clovisdasilvaneto committed:
LPS-84295 Fix rebase conflicts

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

clovisdasilvaneto committed:
LPS-84991 Fix rebase conflicts

clovisdasilvaneto committed:
LPS-84991 Generated snapshots

shuyangzhou committed:
LPS-81743 Store should be a provider type

shuyangzhou committed:
LPS-81743 Use StringBundler

shuyangzhou committed:
LPS-81743 Not needed

drewbrokke committed:
LPS-85261 image-uploader-web - since the overall upload max size may differ from the given maxFileSize, let's keep them separated conceptually

brianchandotcom committed:
LPS-81670 wordsmith

brianchandotcom committed:
LPS-81670 sort

brianchandotcom committed:
LPS-81743 wordsmith

drewbrokke committed:
LPS-85129 users-admin-web - since we're creating our own redirect URL on the edit user page, instead read and pass in a backURL

phcp committed:
LPS-85283 Use DDMFormInstance instead of DDLRecordSet reference on FormStructure API

phcp committed:
LPS-85284 Set DDMFormValues locale on FormValuesUtil

shuyangzhou committed:
LPS-85212 Merge ModuleApplicationContextHolder and ModuleDataSourceFactoryUtil into ModuleDataSourceFactory

phcp committed:
LPS-85285 Include transient fields on Structure representor builder

kiyoshilee committed:
LRQA-43455 Set tomcat gc properties for 7.1.10

Commit:
LPS-84279 Fix poshi test

phcp committed:
LPS-85167 Remove indexType property on FormInstance resource

petershin committed:
LPS-74544 The env variable for liferay.home should be LIFERAY_LIFERAY_PERIOD_HOME

phcp committed:
LPS-85168 Return successPage property only when it's enabled on FormInstance resource

petershin committed:
LPS-74544 Auto SF

phcp committed:
LPS-85169 Remove successPage.enabled property on FormInstance resource

shuyangzhou committed:
LPS-84838 Flip

brianchandotcom committed:
LPS-84838 SF

brianchandotcom committed:
LPS-74544 prep next

brianchandotcom committed:
LPS-74544 prep next

Commit:
LPS-85288 Build Lang

Preston-Crary committed:
LPS-85174 Baseline.

yichenroy committed:
LRQA-43487 Update poshi runner to ver '1.0.169'

javiergamarra committed:
APIO-99 new source format rule

javiergamarra committed:
APIO-99 add missing dependency to prevent conflicts in resolution of services

javiergamarra committed:
APIO-99 sf: sort and don't use wildcard

petershin committed:
LPS-69453 Fix for deployDependencies

petershin committed:
LPS-69453 Update changelog

javiergamarra committed:
APIO-99 sf: new checks for unused code

javiergamarra committed:
APIO-99 fix error of reference caused by sf

yichenroy committed:
LRQA-43487 Build parameters may not always be the first action object

hhuijser committed:
LPS-84119 Exclude dirs from SF

hhuijser committed:
LPS-84119 Remove suppression

hhuijser committed:
LPS-84119 Simplify

brianchandotcom committed:
LPS-85129 Match order in modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/select_regular_role.jspf

brianchandotcom committed:
LPS-69453 prep next

brianchandotcom committed:
Auto SF

brianchandotcom committed:
LPS-84119 prep next

brianchandotcom committed:
LPS-84119 prep next

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 757e391bc38cb7d0954c7b8aba4907468be8d377

Commit:
LPS-84991 SF

Commit:
LPS-84295 SF

gamerson committed:
LPS-84670 rename

ambrinchaudhary committed:
LPS-79515 Add margin top

javiergamarra committed:
APIO-99 remove endpoint bnd reference

jkappler committed:
LPS-85297 Capture exception on adding/editing an asset list

jkappler committed:
LPS-85297 Check session messages accordingly in the Portlet and show error in the view

jkappler committed:
LPS-85297 Adds missing language key

jkappler committed:
LPS-85297 Build lang

pavel-savinov committed:
LPS-85297 Simplify

pavel-savinov committed:
LPS-85297 Add group and guest permissions for the asset list entry

adolfopa committed:
LPS-81513 Avoid extracting text from empty content

In Tika 1.18, this causes an exception to be thrown where previously
it didn't. Add a check for empty content before trying to do anything.

javiergamarra committed:
APIO-99 exclude problematic package

dgarciasarai committed:
LPS-85097 Send notification only if it's not deliver yet

dgarciasarai committed:
LPS-85097 Language key

dgarciasarai committed:
LPS-85097 buildLang

dgarciasarai committed:
LPS-85097 SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/apio-architect/.gitrepo'.

clovisdasilvaneto committed:
LPS-84295 Generated snapshots

clovisdasilvaneto committed:
LPS-84295 Fix tests cases

clovisdasilvaneto committed:
LPS-84295 Generated Builder component snapshots

petershin committed:
LPS-84670 Temporarily disable test

petershin committed:
LPS-65845 Fix writeParentThemesDigest for private themes

petershin committed:
LPS-65845 Use latest gradle-plugins-cache

shuyangzhou committed:
LPS-85329 Detach ClassLoaderComponentConfiguration from ComponentConfiguration

shuyangzhou committed:
LPS-85329 Fail earlier

brianchandotcom committed:
APIO-99 Please fix any more like this

brianchandotcom committed:
APIO-99 What is this?

shuyangzhou committed:
LPS-85329 Refactor ClassLoaderComponentConfiguration to ComponentPropertiesUtil

brianchandotcom committed:
LPS-84119 Fix runtime error. We want to call static 'values' from the enum class, not HashMap#values (which is empty). See https://stackoverflow.com/questions/13659217/where-is-the-documentation-for-the-values-method-of-enum. However, this causes a SF error Reference to method 'values' does not need 'SearchScopePreference.': ./src/main/java/com/liferay/portal/search/web/internal/display/context/SearchScopePreference.java 69 (Checkstyle:SelfReferenceCheck). Hugo SF, I'm fixing it in the next commit but like you to fix the SF tool to allow it in this case.

brianchandotcom committed:
LPS-84119 Hugo SF, please revert this, see previous commit in eb06ccdcbc60fa61e6792c660014745945483d62

shuyangzhou committed:
LPS-85329 SF

javiergamarra committed:
APIO-99 apply optional rule and rename confusing overloaded method name

javiergamarra committed:
APIO-232 renames following conventions

mtambara committed:
LPS-85260 Reconfigure log in json

shuyangzhou committed:
LPS-85329 Fix test

brianchandotcom committed:
LPS-81513 regen

brianchandotcom committed:
LRQA-43487 prep next

brianchandotcom committed:
LPS-65845 prep next

brandizzi committed:
LPS-85286 portal-search-web: Test: by using the same method used in front-end, we ensure it will keep working.

https://issues.liferay.com/browse/LPS-85286

brianchandotcom committed:
LPS-65845 prep next

petershin committed:
LPS-69453 Fix changelog

petershin committed:
LPS-84119 Update changelog

petershin committed:
LPS-74544 Update changelog

petershin committed:
LPS-84119 Update changelog

mtambara committed:
LPS-84999 Only log if there is something to refresh

Commit:
LPS-84295 SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/apio-architect/.gitrepo'.

brianchandotcom committed:
LPS-84670 prep next

brianchandotcom committed:
LPS-84670 Don't add this yet

brianchandotcom committed:
LPS-84670 prep next

brianchandotcom committed:
Revert "LPS-84670 Don't add this yet"

This reverts commit a5082850d2b6256c9c69a8371ffc52aa505f53cb.

brianchandotcom committed:
LPS-84670 prep next

brianchandotcom committed:
LPS-84670 prep next

brianchandotcom committed:
LPS-84670 SF

brianchandotcom committed:
APIO-232 Consistency please

brianchandotcom committed:
APIO-232 Consistency please...

brianchandotcom committed:
APIO-232 Please rename 'f'

brianchandotcom committed:
APIO-232 Short var names hurt in the long run.. make things explicit

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 27c574617dddacfe6fcbf2f05963d7f932a304f0

javiergamarra committed:
APIO-232 Rename bad variable name

rbohl committed:
LRDOCS-5692 remove @review tags for deprecated methods

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/apio-architect/.gitrepo'.

yichenroy committed:
LRQA-43500 Do not append auto-close=false for a SourceFormatBuild

Commit:
LPS-84295 More consistent labels

brianchandotcom committed:
LPS-84295 remove test file

drewbrokke committed:
LPS-84415 user-associated-data-web - main.js - manually destroy the component when Liferay SPA is going to navigate.  That way the interval ajax request does not continue to fire after page navigation

Commit:
LPS-85324 Remove incorrectly translated language keys

Commit:
LPS-85324 Build Lang

brianchandotcom committed:
LRQA-43500 prep next

Commit:
LPS-84478 Add integration tests

hhuijser committed:
LPS-84119 Skip _log

hhuijser committed:
LPS-84119 Fix SF

brianchandotcom committed:
LPS-84119 prep next

brianchandotcom committed:
LPS-84119 prep next

brianchandotcom committed:
LPS-84478 Rename

brianchandotcom committed:
LPS-81743 Fix split package test failure:

junit.framework.AssertionFailedError: Detected split packages in modules/apps/adaptive-media/adaptive-media-image-service and modules/apps/adaptive-media/adaptive-media-image-impl: [com.liferay.adaptive.media.image.internal.util]
	at com.liferay.portal.split.packages.SplitPackagesTest._checkSplitPackages(SplitPackagesTest.java:138)
	at com.liferay.portal.split.packages.SplitPackagesTest.access$000(SplitPackagesTest.java:42)
	at com.liferay.portal.split.packages.SplitPackagesTest$1.preVisitDirectory(SplitPackagesTest.java:95)
	at com.liferay.portal.split.packages.SplitPackagesTest$1.preVisitDirectory(SplitPackagesTest.java:64)
	at java.nio.file.Files.walkFileTree(Files.java:2677)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.split.packages.SplitPackagesTest.testSplitPackage(SplitPackagesTest.java:62)
